src/WebJobs.Script/Diagnostics/OpenTelemetry/TelemetryMode.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Diagnostics.OpenTelemetry
+{
+    internal enum TelemetryMode
+    {
+        ApplicationInsights = 0b0000,
```

Comment:
nit, include a `None` or `Default`. option for enums like this (so you can differentiate between enum not configured and app insights explicitly set.

```suggestion
        None = 0, // or Default
        ApplicationInsights = 1,
```